helm/trento-ai-companion/README.md

```diff
@@ -0,0 +1,61 @@
+<!--
+  ~ Copyright 2025 SUSE LLC
+  ~ SPDX-License-Identifier: Apache-2.0
+-->
+
+# Helm package for trento-ai-companion
+
+<!-- This readme has been created with this tool: https://github.com/bitnami/readme-generator-for-helm
+    > node "./readme-generator-for-helm/bin/index.js" -v ./helm/values.yaml -r ./helm/README.md
+-->
```

Comment:
Don't know if you know this tool to automate the generation of JSON schema and README files for a given Helm chart. It is just a matter of annotating the values.yaml and run the tool. See the results below.

CCing some trento-project/helm-charts contributors: @EMaksy @balanza @nelsonkopliku

Reply:
I didn't know about it, but it looks useful. Thanks for bringing this up!